docs/orchestrating-elastic-stack-applications/security/auth-configs-using-stack-config-policy.asciidoc

```diff
@@ -0,0 +1,20 @@
+:page_id: auth-config-using-stack-config-policy
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{page_id}.html[View this document on the Elastic website]
+****
+endif::[]
+[id="{p}-{page_id}"]
+= Authentication policies using Elastic Stack configuration policy
+
+NOTE: This requires a valid Enterprise license or Enterprise trial license. Check <<{p}-licensing,the license documentation>> for more details about managing licenses.
+
+Starting from ECK `2.11.0` new options were added that enable adding configurations that go into `elasticsearch.yml` and `kibana.yml` to the Elastic Stack configuration policy.
+There is also an option to add additional secrets that will be mounted to the Elasticsearch Pods managed by the Elastic Stack configuration policy. This can be used to configure authorization policies in Elastic Stack configuration policy. 
+The authorization policy will apply to all Elasticsearch clusters and Kibana instances managed by the Elastic Stack configuration policy.
+
+Examples for configuring some of the authentication policies can be found below:
+
+** <<{p}-ldap-using-stack-config-policy,LDAP authentication using Elastic Stack configuration policy>>
+** <<{p}-oidc-stack-config-policy,OpenID Connect authentication using Elastic Stack configuration policy>>
+** <<{p}-jwt-stack-config-policy,JWT authentication using Elastic Stack configuration policy>>
```

Comment:
I wonder if we can do something so that the index structure is more hierarchical
<img width="341" alt="Screenshot 2023-12-18 at 16 08 40" src="https://github.com/elastic/cloud-on-k8s/assets/697790/00da6459-8b99-45f7-9290-fc0487337f2b">


I think what we would like is to have is a structure like 
* Custom HTTP certificate
* Rotate auto-generated credentials
* Managing authentication for multiple stacks using Elastic Stack configuration policy

and then on the right hand side the navigation for the subsections (not sure if that's possible)

Reply:
Let me look into this, this makes sense to me.

Reply:
Made the change
<img width="1553" alt="Screenshot 2023-12-18 at 11 04 44 AM" src="https://github.com/elastic/cloud-on-k8s/assets/135071014/9b285564-44f2-4dd4-98ac-2384cf8ae9d4">